### DISCUSSION TITLE
Help with this error

### DISCUSSION THREAD

**Discussion by andrespisani** [giswqs/geog-312#139] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a7d615d9-5a0e-4bfa-a4f0-81df2890e1a2)

**Comment by giswqs** [comment, marked as answer]
Please make sure you watch the lecture video: https://youtu.be/bLjIlQthUow

You will need to install maplibre. You also need to use Google Colab or JupyterLab. MapLibre does not work with VS Code yet.

**Comment by andrespisani** [comment]
Thank you teacher for the response. I made the first three youtube videos until Intro to GIS Programming | Week 1: Introduction to Conda, Mamba, VS Code, and Git. and when among the comments it gave me this error. But I will follow your advice, thank you very much. I probably did something wrong,

**Comment by andrespisani** [comment]
![image](https://github.com/user-attachments/assets/b6305fcb-7905-41d5-bd61-420c43c1537f)

**Reply by giswqs** [reply]
You need to uncomment the line to install maplibre
```bash
pip install "leafmap[maplibre]"
```

**Reply by andrespisani** [reply]
Thanks, now its working

**Comment by andrespisani** [comment]
Dear professor, thank you very much for your course, it is really good and all software is free. I wanted to ask you since the course is very extensive and takes many hours, I wanted to ask you for a suggestion to focus on a topic that I should go into in your course. My interest is the focus on vulnerable neighborhoods and combining information sources with maps, urban analysis, planning, and presentation of complex data, mobility data, etc.
What do you suggest I do or what courses do you suggest I take?
